### ISSUE TITLE
use system proxy settings

### ISSUE THREAD

**Issue by jli113** [subhra74/xdm#107]
## Checklist

- [ x ] I'm reporting a feature request
- [ x ] I've verified that I'm running xdm latest version
- [ x ] I've searched the bugtracker for similar feature requests including closed ones

## Description

WRITE DESCRIPTION HERE
Can XDM use system proxy like IDM use system settings in proxy/socks configuration?

**Comment by subhra74** [comment]
No, XDM does not use system proxy as off now. Adding this as a feature request. Although you can provide proxy settings in XDM settings
